package.json

```diff
@@ -27,6 +27,7 @@
     "webhook": "./packages/cli/bin/n8n webhook",
     "worker": "./packages/cli/bin/n8n worker",
     "cypress:install": "cypress install",
+    "test:e2e:ui": "cross-env E2E_TESTS=true CYPRESS_BASE_URL=http://localhost:5678 start-server-and-test start http://localhost:5678/favicon.ico 'cypress open'",
```

Comment:
you could remove `CYPRESS_BASE_URL=http://localhost:5678`, that's the current default value